pkg/fission-cli/cmd/environment/command.go

```diff
@@ -59,7 +59,7 @@ func Commands() *cobra.Command {
 		Optional: []flag.Flag{flag.EnvImage, flag.EnvPoolsize,
 			flag.EnvBuilderImage, flag.EnvBuildCmd, flag.EnvImagePullSecret,
 			flag.RunTimeMinCPU, flag.RunTimeMaxCPU, flag.RunTimeMinMemory, flag.RunTimeMaxMemory,
-			flag.EnvTerminationGracePeriod, flag.EnvKeepArchive,
+			flag.EnvTerminationGracePeriod, flag.EnvKeepArchive, flag.EnvRuntime,
```

Comment:
Thanks @AetheWu for the PR. Could you please add support for `flagkey.EnvBuilder` in the same PR as it is present in the create command ?

Reply:
The create command already had the `flagkey.EnvBuilder`, that's why i created the pr.

Reply:
> Thanks @AetheWu for the PR. Could you please add support for `flagkey.EnvBuilder` in the same PR as it is present in the create command ?
The create command already had the `flagkey.EnvBuilder`,  that's why i created the pr.